packages/terra-application/tests/jest/application-loading-overlay/ApplicationLoadingOverlay.test.jsx

```diff
@@ -2,40 +2,27 @@ import React from 'react';
 import { v4 as uuidv4 } from 'uuid';
 
 import ApplicationLoadingOverlay from '../../../src/application-loading-overlay/ApplicationLoadingOverlay';
-import ApplicationLoadingOverlayContext from '../../../src/application-loading-overlay/ApplicationLoadingOverlayContext';
-
-jest.mock('uuid');
 
 describe('ApplicationLoadingOverlay', () => {
-  let reactUseContext;
-  let loadingOverlayContextValue;
+  let mockSpyUuid;
+  let mockSpyContext;
+  const mockUuid = '00000000-0000-0000-0000-000000000000';
```

Comment:
Using a variable can cause scope issues. It's better to hardcode it in the implementation.

Reply:
Please check if this change is what was meant by hardcoding: https://github.com/cerner/terra-application/pull/349/commits/391d232c8a09ff32f83b69f83fabe6fa6f8760ba
It also has been implemented in 2 more places, along with replacing mockImplementation with mockReturnValue: https://github.com/cerner/terra-application/pull/349/commits/5c7579c917f4d40b4b0d35c22753b922d7975045

Reply:
Following up the discussion, has been changed to following: https://github.com/cerner/terra-application/pull/349/commits/e173443ce7964a8c1a5db35eb86b95e1bbc3bca1